src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -407,6 +407,9 @@ public static string CertMustBeRsa(string certificateFriendlyName)
 
         public const string MultiCloudSupportUnavailable = "Multi cloud support unavailable with broker.";
 
+        public const string BrokerAndSsoPolicyConfigurationError = "WithBroker and WithSsoPolicy cannot be used together. Broker by default supports SsoPolicy. Use WithSsoPolicy only when using embedded webview";

```

Comment:
```suggestion
        public const string BrokerAndSsoPolicyConfigurationError = "WithBroker and WithSsoPolicy cannot be used together. Broker by default supports SsoPolicy. Use WithSsoPolicy only when using embedded or system webview";

```

Sorry for going back and forwards on this. I now realize that since you inject this in the "refresh_token" flow, it implicitly covers all webviews.